### PR TITLE
Mhartry

### PR DESCRIPTION
**Pull request by meghan-hartry**
Redid ConnectScene, PatientView, ResultsView, and TestView. Updated the corresponding controllers.
Added settings and connected(phone) icons.
Added gitignore file.

### CODE REVIEW

**Comment by lmonico** [comment]
The GUI updates look good so far. I'll go ahead and check out your branch to look at the views (since I can't read xml lol).

**Comment by lmonico** [comment]
The project was missing a module, so I had to move some things around. I'll commit my fix soon.

**Comment by lmonico** [comment]
Okay so passing a patient object around is simple. We just need to add a data member to controller and pass it in Main. See [https://stackoverflow.com/questions/43429492/how-to-pass-an-object-from-one-controller-to-another-with-javafx](url) it's at the bottom.

File: fieldofvision-pc.iml [hunk 1]
```diff
@@ -10,16 +10,19 @@
     <orderEntry type="module-library" exported="">
```
[the remark this answers is not in the document]
Comment (KelseyCulp) [reply fieldofvision-pc.iml]: I haven't been working on the PC app, but when I move over here I'll let you know if I have this problem

**Comment by KelseyCulp** [comment]
> Okay so passing a patient object around is simple. We just need to add a data member to controller and pass it in Main. See [https://stackoverflow.com/questions/43429492/how-to-pass-an-object-from-one-controller-to-another-with-javafx](url) it's at the bottom.

Good example looks do-able for sure

**Pull request merged**